tests/storage-vm

```diff
@@ -281,6 +281,12 @@ for poolDriver in $poolDriverList; do
         echo "==> Testing VM optimized export/import (while running to check config.mount is excluded)"
         ! lxc export v1 --quiet --compression=none - --optimized-storage | tar -tf - | grep -F config.mount || false
 
+        if [[ $(hasNeededAPIExtension vm_zfs_storage_disk_live_resize) && $(hasNeededAPIExtension vm_lvm_storage_disk_live_resize) && ( "${poolDriver}" == "zfs" || "${poolDriver}" == "lvm" ) ]]; then
+                echo "==> Checking live root disk device resize support with ${poolDriver} backend"
+                lxc config device set v1 root size=16GiB
+                [ "$(($(lxc exec v1 -- blockdev --getsize64 /dev/disk/by-id/scsi-0QEMU_QEMU_HARDDISK_lxd_root) / GiB))" -eq "16" ]
```

Comment:
At that point, I'd also want to double check no resize is scheduled/deferred to next boot.

```
[ "$(lxc config device get v1 volatile.root.apply_quota)" = "" ]
```